ironfish/src/wallet/exporter/encoders/base64json.ts

```diff
@@ -19,31 +19,34 @@ export class Base64JsonEncoder implements AccountEncoder {
   encode(value: AccountImport, options?: AccountEncodingOptions): string {
     const binary = Buffer.from(new JsonEncoder().encode(value))
 
-    if (!options?.encryptWith) {
-      const encoded = binary.toString('base64')
-      return `${BASE64_JSON_ACCOUNT_PREFIX}${encoded}`
-    } else if (isMultisigIdentityEncryption(options.encryptWith)) {
-      const encrypted = encodeEncryptedMultisigAccount(binary, options.encryptWith)
-      const encoded = encrypted.toString('base64')
-      return `${BASE64_JSON_MULTISIG_ENCRYPTED_ACCOUNT_PREFIX}${encoded}`
-    } else {
-      throw new Error('Unknown encryption method requested')
+    if (options?.encryptWith) {
+      if (isMultisigIdentityEncryption(options.encryptWith)) {
+        const encrypted = encodeEncryptedMultisigAccount(binary, options.encryptWith)
+        const encoded = encrypted.toString('base64')
+        return `${BASE64_JSON_MULTISIG_ENCRYPTED_ACCOUNT_PREFIX}${encoded}`
+      } else {
+        throw new Error('Unknown encryption method requested')
+      }
```

Comment:
I think this `else` can be dropped:

```suggestion
      }
      
      throw new Error('Unknown encryption method requested')
```

Reply:
Fixed